README.md

```diff
@@ -23,20 +23,7 @@ of libraries designed to rapidly develop robot applications.
 
 ## Installation
 
-Standard installation can be performed in UNIX systems using the following
-steps:
-
-    mkdir build/
-    cd build/
-    cmake ..
-    sudo make install
-
-## Uninstallation
-
-To uninstall the software installed with the previous steps:
-
-    cd build/
-    sudo make uninstall
+Review the [tutorial section](tutorials/02_installation.md).
```

Comment:
I think this won't work outside of GitHub. This should be the definitive link for that tutorial:

https://ignitionrobotics.org/api/plugin/1.1/installation.html

Reply:
is it the `README.md` used or included in the https://ignitionrobotics.org/api/plugin or other site?

Reply:
It used to show up there, but it broke when we moved to GitHub. We need to fix it, see https://github.com/ignitionrobotics/docs/issues/58